includes/class-wc-payments-token-service.php

```diff
@@ -169,14 +170,12 @@ public function woocommerce_get_customer_payment_tokens( $tokens, $user_id, $gat
 	 * @param WC_Payment_Token $token    Token object.
 	 */
 	public function woocommerce_payment_token_deleted( $token_id, $token ) {
-		if ( WC_Payment_Gateway_WCPay::GATEWAY_ID === $token->get_gateway_id() ) {
-			try {
-				$this->payments_api_client->detach_payment_method( $token->get_token() );
-				// Clear cached payment methods.
-				$this->customer_service->clear_cached_payment_methods_for_user( $token->get_user_id() );
-			} catch ( Exception $e ) {
-				Logger::log( 'Error detaching payment method:' . $e->getMessage() );
-			}
+		try {
```

Comment:
Same thing as above, this function will fire every time a payment token is deleted within WC. Will this function try to delete tokens from other gateways?

Reply:
See the comment above.

Reply:
Verified that other tokens aren't deleted when deleting SEPA tokens. 👍